app/views/layouts/application.html.erb

```diff
@@ -40,7 +40,16 @@
     <%= govuk_header(homepage_url: "/", service_name: t("service.name")) %>
 
     <div class="govuk-width-container">
-      <%= govuk_phase_banner(tag: { text: "Beta" }, text: "This is a new service – your feedback will help us to improve it. ") %>
+      <div class="govuk-phase-banner">
+        <p class="govuk-phase-banner__content">
+          <strong class="govuk-tag govuk-phase-banner__content__tag">
+            Beta
+          </strong>
+          <span class="govuk-phase-banner__text">
+          This is a new service. Help us improve it and <a class="govuk-link" href="/feedback">give your feedback (opens in new tab)</a>.
```

Comment:
This link doesn't open in a new tab.

Reply:
using the govuk-link sets the property to oipen in a new tab to false.  
This may be because of https://support.detectify.com/support/solutions/articles/48001048981-external-links-using-target-blank-